source/bayes_simulation.Rmd

```diff
@@ -91,12 +115,12 @@ requirement of a clearly visualized model is a strong point in favor of simulati
 5. Out of all cars "approved", count how many are "faulty".
    The ratio between these numbers is our answer.
 
-Here is the whole thing:
+Here is the whole thing in code:
 
 ```{python}
 import numpy as np
 
-N = 10000  # number of cars
+N = 10_000  # number of cars
```

Comment:
Could you put a note in the file `notes/general.md` to remind us to introduce the `_` notation in large integers?

Reply:
Done